images/bpftool/build-bpftool.sh

```diff
@@ -14,5 +14,5 @@ make -j "$(getconf _NPROCESSORS_ONLN)"
 
 strip bpftool
 
-mkdir -p /out/linux/amd64/bin
```

Comment:
Admittedly, I don't know how buildx and non-native Docker containers work, but to me it looks like we're no longer compiling both binaries from the same (native) container. Could this use of `TARGETPLATFORM` be dropped since we're effectively performing two separate builds?